source/Tectia/index.rst

```diff
@@ -0,0 +1,589 @@
+* Select "Apply"
+
+
+.. figure:: /images/tectiawin2.png
+
+5. Optional: Set X windows forwarding
+
+Select the "Tunneling" tab.
+
+.. figure:: /images/tectiawin3.png
+
+
+Check the two boxes as illustrated, and click "Apply".
+
+6. In the sidebar menu:
+
+* Select "User Authentication" > "Key Providers" .
+* Select the "Enable Microsoft Crypto API" check box. This is needed to view your CAC card certificates.
+* Select "Apply"
+
+.. figure:: /images/tectiawin4.png
+
+6. Set up a connection profile for each hostname that you want to use. There are two bastions, one in Boulder, CO and one in Princeton, NJ. It is highly recommended that you set up a profile from each bastion for each RDHPCS system you need to use, as bastions are typically down during maintenance periods.
+
+* In the sidebar menu select "Connection Profiles".
+* Select "Add Profile".
+* In the "Connection" tab: Fill out the information for the hostname you are configuring. * Enter the "Profile Name" you want to assign to the hostname (ex: Jet-BLDR bastion). Leave "Port number" =22.
+* Enter the Host Name from the bastion list. 
+* Select "Apply".
+* To add another profile select "Add Profile" in the lower left, and repeat the above steps.
+* Select "OK" when all profiles are set.
+
+The example below shows a profile for the CAC Gaea bastion in Princeton. The port used (22) is correct, as is the User Name selection. This can be set here, or just select the radio button next to “Use the Default Connection’s username”.
+
+.. figure:: /images/tectiawin5.png
+
+**Set Up Port Tunnelling**
+
+Complete the following sequence to set up port tunnelling.
+
+1. Edit your connection profile. Navigate to the "Tunneling" tab.
+
+.. figure:: /images/tectiawin6.png
+
+2. Check "Use Defaults". Tunnel X11 connections" and "Allow Agent Forwarding" should be checked. If not, check them. 
+
+.. figure:: /images/tectiawin7.png
+
+3. Select "Add".
+
+* Select "TCP" for Type
+* Listen Port should match your Local port number listed on your session login.
+* Check "Allow local connections only"
+* Destination host: 127.0.0.1
+* Destination Port should match your Local port number listed on your session login.
+
+.. figure:: /images/tectiawin8.png
+
+* Select "OK"
+
+4. Selecting "Test connection" to test. 
+
+.. figure:: /images/tectiawin9.png
+
+* Completed configuration should look like the following:
+
+.. figure:: /images/tectiawin10.png
+
+Once the session is open, you will be able to use this forwarded port for data transfers as long as this ssh window is kept open.
+After the first session has been opened with the port forwarding, any further connections (login via ssh, copy via scp) will work as expected. 
+
+
+Configuring Tectia:  RHEL
+-------------------------
+
+**Requirements**
+
+Tectia SSH will install on the following RHEL/CentOS version:
+
+.. code:: shell
+
+    RHEL/CentOS (x86)
```

Comment:
Both of these versions are outdated but I'm not sure quite what to set them too.  Maybe just replace 6/7 with 7/8.

Reply:
done